app/models/ingest_job.rb

```diff
@@ -391,6 +391,13 @@ def set_study_state_after_ingest
       create_differential_expression_results
     when :image_pipeline
       set_has_image_cache
+    when :ingest_anndata
+      # currently extracting and ingesting only clustering data
+      # this will likely error until the DB inserts ingest job is done
+      set_cluster_point_count
+      set_study_default_options
+      launch_subsample_jobs
+      # TODO (SCP-4708)(SCP-4709)(SCP-4710) will duplicate a lot more from above 
```

Comment:
Like we delimited TODO tickets elsewhere in code, and in PR titles:

```suggestion
      # TODO (SCP-4708, SCP-4709, SCP-4710) will duplicate a lot more from above 
```